src/js/evented.js

```diff
@@ -1,15 +1,13 @@
-export class Evented {
-  constructor(/* options = {}*/) {
-    // TODO: do we need this empty constructor?
-  }
+import _ from 'lodash';
 
+export class Evented {
   on(event, handler, ctx) {
     const once = arguments.length <= 3 || arguments[3] === undefined ? false : arguments[3];
 
-    if (typeof this.bindings === 'undefined') {
+    if (_.isUndefined(this.bindings)) {
```

Comment:
Think the original intent is to check if it's an object, perhaps that one instead?